docs/setup-aws-lambda.asciidoc

```diff
@@ -1,85 +1,62 @@
 [[aws-lambda]]
-=== Get started with Lambda
+=== Monitoring AWS Lambda Java Functions
+:layer-section-type: with-agent
 
-Getting Elastic APM set up for your lambda functions is easy,
-and there are various ways you can tweak it to fit your needs.
+The Java APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
 
-NOTE: In order to get the full AWS Lambda tracing capabilities, use with APM Server 7.16 or higher. Using
-older versions provides most value, but some relevant metadata fields will not be indexed.
+Follow the <<aws-lambda-java-quick-start>> Guide to get started in minutes and checkout the <<aws-lambda-features-and-caveats>> to learn about the features and limitations of the Java APM Agent on AWS Lambda Functions.
 
 [float]
-[[aws-lambda-runtimes]]
-==== Supported runtimes
+[[aws-lambda-java-quick-start]]
+==== Quick Start
 
-AWS Lambda provides multiple https://docs.aws.amazon.com/lambda/latest/dg/java-image.html[JVM base images], only the ones that support the `AWS_LAMBDA_EXEC_WRAPPER` environment variables
-are supported out of the box.
+include::{apm-aws-lambda-root}/docs/add-extension/code.asciidoc[]
 
-Running with unsupported images is still possible but requires providing agent configuration through environment variables
-explicitly.
+Before starting with the setup make sure you are using one of the  <<supported-aws-lambda-runtimes, supported AWS Lambda runtimes>>.
```

Comment:
As this is a prerequisite to getting started, we should wrap the `AWS Lambda runtimes` section (excluding the heading) in a tagged region and include it on this page. This will single-source the content and prevent users from having to navigate away from this page to get started.